### ISSUE TITLE
num_workers 是怎么设置

### ISSUE THREAD

**Issue by huyuanchao** [PaddlePaddle/PaddleSlim#1490]
train_loader = paddle.io.DataLoader(dataset, batch_size=32, shuffle=True, drop_last=True, num_workers=4)
         
val_loader = paddle.io.DataLoader(dataset, batch_size=32, shuffle=False, drop_last=True, num_workers=4)

我用同服务器的8张卡训练和验证，是否需要指定num_workers，需要设置成多少？

**Comment by wanghaoshuang** [comment]
https://www.paddlepaddle.org.cn/documentation/docs/zh/2.4rc/api/paddle/io/DataLoader_cn.html#dataloader
num_workers用于加速数据读取，并不影响训练逻辑
